src/main/kotlin/com/safehill/kclient/controllers/UserInteractionController.kt

```diff
@@ -100,8 +100,9 @@ class UserInteractionController internal constructor(
         return if (existingThread != null) {
```

Comment:
Are we checking existingThread existence on remote?

Reply:
yeah.. the serverProxy first checks the local server and then the remote server.

Reply:
Should it be the other way around? Check the remote, and fall back to local if there is no connection?
I believe that's what's happening in Safehill-Client

Reply:
Yeah, I agree... Previously the thread's members could not be changed but now we should definitely check the remote server first. I had plans to talk about this in the meeting but forgot it.. I am glad that you caught it up.